### PR TITLE
chore: preview isn't automatic

### PR DESCRIPTION
**Pull request by krshdev**
The preview isn't automatic, this makes it automatic

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/good-first-issue-finder/pull/38"></a>

### CODE REVIEW

**Comment by Cahllagerfeld** [comment]
Thats intentional, because we can't run the app directly when starting. We probably need to even remove the `npm run dev` from the yml-file, as we need to provide an `env` first

**Comment by krshdev** [comment]
I think we can do that in gitpod itself, there is a prompt which asks you, as said in the gitpodify blog 🤔

**Comment by Cahllagerfeld** [comment]
I think this PR adds way too much overhead, which isn't necessary, even a new dependency.
What we should just do is removing the automatic start from `command` in the `gitpod.yml`
```diff
tasks:
  - init: npm install # runs during prebuild
    command: |
      export HMR_HOST=`gp url 3000`
-      npm run dev
ports:
  - port: 3000
    onOpen: ignore

github:
  prebuilds:
    master: true
    branches: true
    pullRequests: true
    pullRequestsFromForks: true
    addCheck: true
    addComment: false
    addBadge: true

vscode:
  extensions:
    - humao.rest-client
    - svelte.svelte-vscode
    - bradlc.vscode-tailwindcss

```

File: .gitpod.yml [hunk 1]
```diff
@@ -2,11 +2,11 @@ tasks:
   - init: npm install # runs during prebuild
     command: |
       export HMR_HOST=`gp url 3000`
-      npm run dev
+      npm run gitpod
```
Comment (Siddhant-K-code) [inline .gitpod.yml:5]:
```suggestion
      npm run dev
```

will work fine.

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]
- `npm run gitpod`  does not required.
- regarding other file changes, pls add the more description about those changes.

File: .gitpod.yml (hunk 1 above)
Comment (krshdev) [reply .gitpod.yml:5]: This isn't how it has to be fixed 😅

File: package-lock.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "issue-finder",
-  "version": "0.3.5",
+  "name": "issue-crawler",
+  "version": "0.0.1",
```
Comment (Cahllagerfeld) [inline package-lock.json:3]: theres a newer lockfile already, please update yours or remove it from the commit 👍

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]

**Review by Cahllagerfeld — APPROVED** [review]
Awesome, thank you 💥

**Pull request merged**